tests/test_helpers.py

```diff
@@ -941,3 +942,38 @@ def test_safe_join_exceptions(self):
         for args in failing:
             with pytest.raises(NotFound):
                 print(flask.safe_join(*args))
+
+class TestHelpers(object):
+
+    def test_get_debug_flag(self, monkeypatch):
+        monkeypatch.setenv('FLASK_DEBUG', '')
+        assert get_debug_flag() == None
```

Comment:
Good call. I did add some logic around this after parameterizing (not sure if that was overkill in this case).